### PR TITLE
User Management

### PR DESCRIPTION
**Pull request by katieann7**

### CODE REVIEW

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
remove integrated from the project name

**Comment by katieann7** [comment]
Done po sir

File: user-management-application/build.xml [hunk 1]
```diff
@@ -1,36 +1,34 @@
...
-    <property name="doc.dir" value="build/doc"/>
-
-    <target name="clean">
-        <delete dir="${build.dir}"/>
-    </target>
-
-    <target name="compile">
-        <mkdir dir="${classes.dir}"/>
-        <javac srcdir="${src.dir}" destdir="${classes.dir}"/>
+<project name="Ant Test Project" default="build">
```
Comment (RoCS2024) [inline user-management-application/build.xml:2]: project name is Ant Test Project

File: user-management-application/ivy.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ivy-module version="2.0"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:noNamespaceSchemaLocation="http://ant.apache.org/ivy/schemas/ivy.xsd">
+    <info
+            organisation="com.rogationist.college"
+            module="ivy-test"
+            status="integration">
+    </info>
+    <dependencies>
```
Comment (RoCS2024) [inline user-management-application/ivy.xml:10]: how about unit test jars?

File: user-management-application/src/main/java/com/user/management/data/connection/ConnectionHelper.java [hunk 3]
```diff
@@ -1,27 +1,4 @@
-/**
- * This is a connection helper class.
- * this class is responsible in establishing a connection with the database.
- * this connection helper is connected to an Oracle Database.
- * */
-
 package com.user.management.data.connection;
```
Comment (RoCS2024) [inline user-management-application/src/main/java/com/user/management/data/connection/ConnectionHelper.java:1]: why also delete ConnectionHelper?

File: user-management-application/src/main/java/com/user/management/data/user/dao/impl/UserDaoImpl.java [hunk 4]
```diff
@@ -1,59 +1,4 @@
...
-            stmt.setInt(5, user.getId());
-            int affectedRows = stmt.executeUpdate();
-            return affectedRows > 0;
-        } catch (SQLException ex) {
-            System.err.println("Error updating user with ID " + user.getId() + ": " + ex.getMessage());
-            ex.printStackTrace();
-            return false;
-        }
-    }
+public class UserDaoImpl {
```
Comment (RoCS2024) [inline user-management-application/src/main/java/com/user/management/data/user/dao/impl/UserDaoImpl.java:3]: where's implements UserDao?

File: user-management-application/src/main/java/com/user/management/app/facade/user/impl/UserFacadeImpl.java [hunk 5]
```diff
@@ -1,38 +1,4 @@
...
-            if(targetUser == null) {
-                throw new Exception("Customer to update not found. ");
-            }
-            result = userDao.updateUser(user);
-        } catch (Exception e) {
-            throw new RuntimeException(e.getMessage());
-        }
-        return result;
-    }
+public class UserFacadeImpl {
```
Comment (RoCS2024) [inline user-management-application/src/main/java/com/user/management/app/facade/user/impl/UserFacadeImpl.java:3]: where is implements UserFacade?

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
see comments

**Review by JustineDaveMagnaye — APPROVED** [review]

**Comment by katieann7** [comment]
Done na po sir

**Review by RoCS2024 — APPROVED** [review]
all good.

**Pull request merged**